alembic/env.py

```diff
@@ -0,0 +1,81 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from city.models import *
+from temperature.models import *
```

Comment:
Ensure that the `Base` class is imported from the appropriate module where it is defined. This is necessary for `target_metadata = Base.metadata` to work correctly. Check the `database.py` file or wherever the `Base` is declared and import it here.